api/app/reviews/models.py

```diff
@@ -0,0 +1,103 @@
+from datetime import datetime
+
+from app import db
+
+class ReviewForm(db.Model):
+    id = db.Column(db.Integer(), primary_key=True)
+    application_form_id = db.Column(db.Integer(), db.ForeignKey('application_form.id'), nullable=False)
+    is_open = db.Column(db.Boolean(), nullable=False)
+    deadline = db.Column(db.DateTime(), nullable=False)
+
+    application_form = db.relationship('ApplicationForm', foreign_keys=[application_form_id])
+
+    def __init__(self, application_form_id, deadline):
+        self.application_form_id = application_form_id
+        self.is_open = True
+        self.deadline = deadline
+
+    def close(self):
+        self.is_open = False
+
+
+class ReviewQuestion(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    review_form_id = db.Column(db.Integer(), db.ForeignKey('review_form.id'), nullable=False)
+    question_id = db.Column(db.Integer(), db.ForeignKey('question.id'), nullable=False)

```

Comment:
This should be nullable, because some ReviewQuestions will not be linked to an application form question

Reply:
Done.